prefect_databricks/rest/_generated.py

```diff
@@ -93,29 +93,6 @@ async def execute_endpoint(
 
     Returns:
         The httpx.Response from interacting with the endpoint.
-
-    Examples:
-        Lists jobs on the Databricks instance.
-        ```python
-        from prefect import flow
-        from prefect_databricks import DatabricksCredentials
-        from prefect_databricks.rest import execute_endpoint
-        @flow
-        def example_execute_endpoint_flow():
-            endpoint = "/2.1/jobs/list"
-            databricks_credentials = DatabricksCredentials.load("my-block")
-            params = {
-                "limit": 5,
-                "offset": None,
-                "expand_tasks": True,
-            }
-            response = execute_endpoint(
-                endpoint,
-                databricks_credentials,
-                params=params
-            )
-            return response.json()
```

Comment:
Why was this example removed?

Reply:
It previously overrided the existing example, but with openapi-python-client this is no longer a file.